src/sage/categories/simplicial_sets.py

```diff
@@ -8,13 +8,22 @@
 #                  https://www.gnu.org/licenses/
 # *****************************************************************************
 
+from sage.functions.generalized import sign
+from sage.matrix.special import identity_matrix
 from sage.misc.cachefunc import cached_method
+from sage.misc.lazy_import import lazy_import
 from sage.categories.category_singleton import Category_singleton
 from sage.categories.category_with_axiom import CategoryWithAxiom
 from sage.categories.sets_cat import Sets
 from sage.categories.homsets import HomsetsCategory
+from sage.matrix.constructor import matrix
+from sage.misc.flatten import flatten
```

Comment:
I think this line can be deleted: I don't see `flatten` anywhere in the rest of this file.